azure-devops/projects/cgn-onboarding-portal-projects/cgn-onboarding-portal-frontend.tf

```
@@ -0,0 +1,154 @@
+variable "cgn-onboarding-portal-frontend" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "cgn-onboarding-portal-frontend"
+      branch_name    = "master"
+      pipelines_path = ".devops"
+    }
+    pipeline = {
+      # TODO
+      production_storage_account_name = ""
+      blob_container_name             = ""
+      staging_storage_account_name    = ""
+      cache_version_id                = "v1"
+    }
+  }
+}
+
+# code review
+resource "azuredevops_build_definition" "cgn-onboarding-portal-frontend-code-review" {
+  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-ro, azuredevops_project.project]
```

```suggestion
  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-pr, azuredevops_project.project]
```

We need `io-azure-devops-github-pr` because this connection has the permission to create webhook on github repo to trigger pipelines on pr